modules/pel/peltool/README.md

```diff
@@ -0,0 +1,72 @@
+### peltool Usage
+To get more peltool's capabilities, use the help option `peltool.py --help`.
+
+**Note:** For non-BMC environment user must pass the PEL directory to parse PELs
+          e.g. `peltool.py -p <pel_dir> -l`, `peltool.py -p <pel_dir> -n`, `peltool.py -p <pel_dir> -a`
+
+#### List PELs
+- Get list of PELs : `peltool.py -l` or `peltool.py -ls` (By default serviceable PELs are listed)
+- Get list of PELs including serviceable PELs : `peltool.py -ls`
```

Comment:
you already had this on line 8?  i think you can delete this line since there isn't a need to ever use -ls.